driver/test/column_bindings_it.cpp

```diff
@@ -0,0 +1,487 @@
+#include "driver/platform/platform.h"
+#include "driver/test/client_utils.h"
+#include "driver/test/client_test_base.h"
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+class ColumnBindingsTest
+    : public ClientTestBase
+{
+};
+
+class ColumnArrayBindingsTest
+    : public ColumnBindingsTest
+    , public ::testing::WithParamInterface<std::tuple<std::uintptr_t, std::size_t, std::size_t>>
+{
+};
+
+template <typename CharType, std::size_t Size>
+struct FixedStringBuffer {
+    FixedStringBuffer() {
+        // Fill with a special placeholder sequence.
+        std::fill(data, data + Size, 0b11011011);
+    }
+
+    CharType data[Size];
+};
+
+TEST_P(ColumnArrayBindingsTest, ColumnWise) {
+    const std::size_t total_rows_expected = std::get<1>(GetParam());
```

Comment:
Could you please outline the test purpose and major points in the function-level comment, something like:
Bind various columns of fixed size, emulate a query, and validate that indices/offsets are valid.

Otherwise, it is really hard to decode your intent.

Reply:
Added comments.